packages/dds/tree/tsconfig.cjs.json

```diff
@@ -0,0 +1,17 @@
+{
+	"extends": "./tsconfig.base.json",
+	"compilerOptions": {
+		"outDir": "./dist",
+		// Ideally there would be no CommonJS transpilation and wrappers would exist around
+		// ESM modules as needed. Until there is such support also transpile to CommonJS.
+		// Next preference is to use Node16 for module and moduleResolution with package.json
+		// "type": "commonjs". However "type": "module" is required to activate full ESM Mode
+		// for the ESM build.
+		// If solution like tsc-multi adds support for package.json type override then this
+		// can be Node16 with a "type": "commonjs" package.json override.
+		// Underlying reason for the complexity here is that base tools wish to avoid the dual
+		// package hazard of ESM and CommonJS in the same package. "tsc-multi" is a workaround.
+		"module": "CommonJS",
+		"moduleResolution": "Node10",
```

Comment:
Are we certain this produces valid output?

Reply:
Yes. This is not a change from what tree was setup for before. moduleResolution was node which is the deprecated specification for Node10.